### PR TITLE
jsk_data: download_data.py: ensure chmod downloaded data if possible

### PR DESCRIPTION
**Pull request by furushchev**
- Fix: Make sure to change permission even if downloading is interrupted
- Fix: Check file permission before `chmod` since `os.chmod` fails when changing the permission of files which were already changed.

### CODE REVIEW

File: jsk_data/src/jsk_data/download_data.py [hunk 1]
```diff
@@ -52,11 +63,15 @@ def decompress_rosbag(path, quiet=False, chmod=True):
...
-        os.chmod(orig_path, 0777)
-        os.chmod(path, 0777)
+    try:
+        rosbag.rosbag_main.decompress_cmd(argv)
+    finally:
+        if chmod:
+            if not is_file_writable(path):
+                os.chmod(path, 0777)
+            orig_path = osp.splitext(path)[0] + '.orig.bag'
+            if not is_file_writable(path):
```
Comment (wkentaro) [inline jsk_data/src/jsk_data/download_data.py]: Maybe path -> orig_path?

Comment (furushchev) [reply jsk_data/src/jsk_data/download_data.py]: @wkentaro nice catch!

**Comment by furushchev** [comment]
@wkentaro Any other comments?

**Comment by furushchev** [comment]
@wkentaro Thanks!
@k-okada Please merge if it's ok for you! :+1:

**Comment by furushchev** [comment]
@k-okada Please merge this if there is no problem, since without this pull request building `jsk_recognition` packages with shared downloaded data fails in some environments.

**Comment by furushchev** [comment]
wait for #1573

**Pull request merged**